README.md

```diff
@@ -1 +1,22 @@
-# rmb-sdk-ts
\ No newline at end of file
+# Introduction
+This is a `Typescript` sdk that can be used to build both **services**, and **clients**
+that can talk over the `rmb`.
+
+[RMB](https://github.com/threefoldtech/rmb-rs) is a message bus that enable secure
+and reliable `RPC` calls across the globe.
+
+`RMB` itself does not implement an RPC protocol, but just the secure and reliable messaging
+hence it's up to server and client to implement their own data format.
+
+## Pre Requirements
```

Comment:
That's too complicated already, the end user shouldn't be bothered by deploying a relay and running it, please link to one of the deployed relay and move that part of running your own relay to a separate page or flag it as advanced, but definitely not a prerequisite

Reply:
done